src/data/registry.xml

```diff
@@ -491,5 +491,76 @@
       <long_name>graupel mass mixing ratio with respect to moist air plus all airborne condensates</long_name>
       <ic_file_input_names>GRAUQM cnst_GRAUQM</ic_file_input_names>
     </variable>
+
+    <!-- cam_in Variables -->
+    <variable local_name="landfrac"
+              standard_name="land_area_fraction"
+              units="fraction" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <ic_file_input_names>landfrac cam_in_landfrac</ic_file_input_names>
+    </variable>
+
+    <!-- Zhang McFarlane (ZM) Variables -->
+    <!-- Note that no_deep_pbl defaults to false, but needs to be set to true for diag_TKE or UW schemes -->
+    <variable local_name="pblh"
+              standard_name="atmosphere_boundary_layer_thickness"
+              units="m" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <ic_file_input_names>pblh pbuf_pblh</ic_file_input_names>
+    </variable>
+    <variable local_name="delt"
+              standard_name="half_timestep_for_physics"
+              units="s" type="real" kind="kind_phys">
+    </variable>
+    <variable local_name="tpert"
+              standard_name="convective_temperature_perturbation_due_to_pbl_eddies"
+              units="K" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <ic_file_input_names>tpert pbuf_tpert</ic_file_input_names>
+    </variable>
+    <variable local_name="ql"
```

Comment:
Could I ask why the local name in SIMA is set to `ql` instead of `icwmrdp` as in the snapshot? I was not totally sure that `icwmrdp` corresponds exactly to `ql` since there is a `icwmrsh` which is the shallow convection equivalent, but I couldn't find a clear definition of the `ql` in the current CAM's state to be sure.

Reply:
The simple answer is that that is what the initial developer (whoever that was) used it in their code.  Inside zm_convr_intr.F90 which resides in ESCOMP/CAM, there is a pbuf call `call pbuf_get_field(pbuf, icwmrdp_idx,     ql )`.  That links the two variable names together.

Reply:
Changed as requested